assets/css/style.scss

```diff
@@ -133,3 +133,8 @@ span.wc-apa-amazon-logo {
 #pay_with_amazon_product {
 	margin: 0.7em auto;
 }
+
+.site-header .widget_shopping_cart div.buttons {
```

Comment:
Not a blocker, just a quick question.
Imho we are setting the same padding than storefront theme, what about other themes?